textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/dictionarymatcher/DictionaryMatcherTest.java

```diff
@@ -53,7 +53,7 @@
     private IDataWriter dataWriter;
     private IDataReader dataReader;
     private Analyzer analyzer;
```

Comment:
I was actually referring to the class `DictionaryMatcher` which is not shown in this PR since it's not modified.  Can we move some of its class-level variables?